keyboards/acheron/doddle60/keymaps/via/rules.mk

```
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
+BOOTMAGIC_ENABLE = yes
```

Given the via specific handling (and the fact you could bork your keymap), best to use lite here.
```suggestion
BOOTMAGIC_ENABLE = lite
```